torch/testing/_internal/distributed/rpc_utils.py

```diff
@@ -48,6 +48,7 @@
 from torch.testing._internal.distributed.rpc.examples.reinforcement_learning_rpc_test import (
     ReinforcementLearningRpcTest,
 )
+from torch.testing._internal.distributed.rpc.cuda_distributed_rpc_test import CudaDistributedRPCTest
```

Comment:
Can we put this file under `torch.testing._internal.distributed.rpc.examples` folder instead?

Reply:
okay moved it.